documentation/monitoring.md

```diff
@@ -75,6 +75,44 @@ There are several other variables that can be changed depending on env requireme
 - grafana_app_mem - app memory limit (default 1Gi)
 - grafana_app_cpu - app requests cpu (default 500m)
 
+Grafana could be accessed via the respective URL's based on the environment of interest. The URL's corresponding to each environment is as below
+
+### Test | https://grafana.test.teacherservices.cloud 
+### Production | https://grafana.teacherservices.cloud/
+
+The default access to the grafana interface is view only, which does not require authentication. In order to be able to make changes for example adding more dashboards and editing existing dashboards, requests will have to be made in the Teacher-services-Infra slack channel to obtain admin credentials 
+
+### Editing or creating new grafana dashboards
+The following steps are required for creating or editing dashboards
```

Comment:
Can we link to the doc? https://grafana.com/docs/grafana/latest/fundamentals/dashboards-overview/